pyproject.toml

```
@@ -26,6 +26,13 @@ ruff = "==0.0.290"
 tomli = "^2.0.1"
 typing_extensions = ">=4.4.0"
 
+[tool.poetry.group.sql.dependencies]
+great-expectations = { version = "^0.17.15", extras = [
+    "postgresql",
+    "redshift",
+    "sqlalchemy",
+] }
```

```suggestion
great-expectations = { version = "^0.17.15", extras = [
    "snowflake",
] }
```

Let's focus on snowflake first.
Try to lock/resolve the dependencies with just `snowflake`.